apps/studio/src/server/modules/auth/singpass/singpass.router.ts

```diff
@@ -55,6 +55,32 @@ export const singpassRouter = router({
       }
     }),
 
+  getName: publicProcedure.query(async ({ ctx }) => {
```

Comment:
might be misleading since it returns an object of both `name` and `isNewUser`

Reply:
Updated in upstream.